netbox_onboarding/metrics.py

```diff
@@ -0,0 +1,23 @@
+from prometheus_client.core import GaugeMetricFamily
+
+
+def ot_exec_time():
+    """Export onboarding task time related metrics."""
+    from .models import OnboardingTask  # pylint: disable=import-outside-toplevel
+
+    ots = OnboardingTask.objects.filter(onboarding_start__isnull=False, onboarding_stop__isnull=False,)
+
+    g = GaugeMetricFamily("ot_exec_time", "time (s) it takes to execute onboarding task", labels=["type"])
```

Comment:
This might be better presented as a [histogram or summary](https://prometheus.io/docs/practices/histograms/) rather than a min/max/avg set of gauges. (Full disclosure: I don't understand Prometheus histograms/summaries very well myself and would love to see a worked example here to learn from :-) )